meter/subscription_manager.cc

```diff
@@ -268,7 +268,7 @@ void SubscriptionManager::Stop() noexcept {
 
     flush_metrics();
     Logger()->info("Advancing clock and flushing metrics");
-    clock_->SetOffset(4990);
+    clock_->SetOffset(59990);
```

Comment:
Why is it 10ms less than the step? Should this come from a configuration setting?

Reply:
The idea was to avoid skipping one minute, but it was just too hacky. Updated the PR with a better method of calculating the offset needed to send metrics with the next step.